Dockerfile.special-resource-operator-build

```
@@ -6,12 +6,11 @@ ENV GOFLAGS=""
 RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
 RUN chmod 700 get_helm.sh
 RUN ./get_helm.sh
-RUN yum install -y which
-RUN yum install -y podman docker
+RUN yum install -y podman docker which
 RUN curl -L "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" -o /tmp/kubectl
 RUN install -o root -g root -m 0755 /tmp/kubectl /usr/local/bin/kubectl
-RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.42.1
-RUN GO111MODULE=on go install golang.stackrox.io/kube-linter/cmd/kube-linter@latest
+RUN go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.33.0
+RUN go install golang.stackrox.io/kube-linter/cmd/kube-linter@v0.0.0-20210328011908-cb34f2cc447f
 RUN curl -L --retry 5 "https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2Fv4.3.0/kustomize_v4.3.0_linux_amd64.tar.gz" | \
     tar -zx -C /usr/bin
 RUN go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.6.0
```

How about using the existing Makefile targets here instead of doing this manually? This removes the risk of a version mismatch between the Makefile and the Skipper Dockerfile.

```suggestion
RUN make controller-gen golangci-lint kube-linter kustomize
```

---

Isn't it creating a loop?

user run skipper with some makefile command --> skipper call makefile

doesn't feel right

---

But this is to build the image in which Skipper commands will be ran later - I don't see a loop here.

---

When building skipper image, the code directory is not mapped, so we cannot run Makefile commands. I think that a more correct solution is to have a install script, that can be called from the makefile target and can be copied to the skipper Dockerfile during build and executed. For now, i prefer to leave it as is and see if the team will be comfortable in working with skipper ( skipper code does not interfere with the usual Makefile commands